src/components/card/ItemPagination.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Pagination, PaginationLink, PaginationItem } from "reactstrap";
+
+const ItemPagination = ({
+  itemsPerPage,
```

Comment:
It's a good habit to set default values to reduce any chance of the math calculations below failing due to a value potentially being undefined